### ISSUE TITLE
buildifier: **kwargs trailing comma invalid

### ISSUE THREAD

**Issue by hsyed** [bazelbuild/buildtools#292]
I am working on grpc stub generator for kotlin coroutines. I adapted the rule from grpc-java which contains the following block:

```
    _gensource(
        name = gensource_name,
        srcs = srcs,
        flavor = flavor,
        enable_deprecated = enable_deprecated,
        visibility = ["//visibility:private"],
        **kwargs
    )
```

Running this through buildifier causes a trailing command for kwargs which is invalid skylark. 

---

Additionally is there a first line comment we could add to disable buildifier processing (similar to go gazelle) ? Should I create a new issue ?

**Comment by andyscott** [comment]
Nice! I just opened a PR to fix this because I ran into the same problem over this last week.
https://github.com/bazelbuild/buildtools/pull/293

**Comment by vladmos** [comment]
Looks like fixed after #293 had been merged.
